spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -467,4 +480,135 @@ class PerfRunHelperClass
       subject.copy_system_logs(myhost)
     end
   end
+  describe "#validate_results_to_baseline" do
```

Comment:
:pick: the spec tests in this file use both ".mymethod" and "#mymethod" syntax. Should we standardize? If so, which wins?

Reply:
Per http://www.betterspecs.org/#describe:

> ...use the Ruby documentation convention of . (or ::) when referring to a class method's name and # when referring to an instance method's name.